guidelines/terms/22/target-offset.html

```diff
@@ -1,6 +1,11 @@
 <dt class="new"><dfn data-lt="target offsets">target offset</dfn></dt>
 <dd class="new">
 	<p class="change">New</p>	
-   <p>the distance measured from the farthest point of a target (A), to the nearest point of the second target (B). The offset includes the target and spacing around the target.</p>
+   <p>size of a target (A) plus the spacing to a second target (B).</p>
    <p class="note">The offset from A to B may be different than the offset from B to A, if the sizes of these targets differ.</p>
+   <div class="note">
+      <p>For horizontally aligned targets, target offset is the width of target A plus the horizontal space to the closest edge of target B. For vertically aligned targets, target offset is the the height of target A plus the vertical space to the closest edge of target B. For targets that are neither horizontally nor vertically aligned, the target offset is the longest measure from any point of A to the nearest point of B.</p>
```

Comment:
How about:

```suggestion
      <p>Offset is the length of the longest possible line that starts at an edge of A, intersects a second edge of A, and ends at an edge of B. For horizontally aligned targets, offset is measured with a horizontal line. For vertically aligned targets, offset is measured with a vertical line. For targets that are not aligned the offset is measured horizontal, vertical or diagonal (whichever is longest).</p>
```

Reply:
I think "and ends at an edge of B" needs to be "and ends at the closest edge of B", otherwise you could extend it to the furthest edge of B.

Also, if a target is not aligned vertically or horizontally, that will have to be diagonal won't it? So the last bit could be "For targets that are not aligned the offset is measured diagonally".